{
  "_annotation": {
    "cve_id": "CVE-2024-45847",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-45847.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:mindsdb:mindsdb:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "E96FCAC6-EFFC-5CDB-91AB-C0B6384B3C00",
                "versionEndExcluding": "24.7.4.1",
                "versionStartIncluding": "23.11.4.2",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}